cobalt/site/docs/development/setup-android.md

```diff
@@ -151,81 +187,113 @@ Go ahead and click 'yes' to open the SDK manager to install the following:
+
+1. Build nplb library
+
+    ```sh
+    ninja -C out/evergreen-arm-softfp_devel nplb_install
     ```
 
-1.  To add command line parameters add `--esa` to specify the "args" array as
-    comma-separated values (with characters like '&' backslash-escaped to make
-    it through the launch command) under:
+1. Generate apk output folder
 
-    Run -> Edit Configurations… -> "app" -> General -> Launch Options -> Launch Flags
+    ```sh
+    gn gen out/android-arm_devel --args="target_platform=\"android-arm\" target_cpu=\"arm\" target_os=\"android\" sb_is_evergreen_compatible=true build_type=\"devel\" sb_api_version=15"
+    ```
 
-    e.g. To run with a different URL: `--esa args --url=<DIFFERENT_URL>`
-1.  To monitor log output, see the `Logcat` on the bottom-left of the IDE. You
-    can enter "starboard" in the search bubble to filter the output.
-1.  To kill the app go to Run -> Stop, or click the red square stop button
-    either on the top toolbar, or in the debugger on the bottom-left of the IDE.
-1.  To set breakpoints in native code, just open the relevant source file with
-    File -> Open… (sorry, files outside the apk project can't appear in the
-    Project panel of the IDE) and click in the gutter of the relevant line.
-    (Once you have one C++ file open, the path breadcrumbs at the top are useful
-    to open other nearby files.)
+### Build and run nplb apk for evergreen
 
+1. Build nplb apk
 
-## Running Tests
+    ```sh
+    ninja -C out/android-arm_devel nplb_evergreen_loader_install
+    ```
 
-The test target itself (e.g. nplb) just builds an .so file (e.g. libnplb.so). To
-run that on a device, it needs to be packaged into an APK, which is done by the
-associated "install" target (e.g. nplb_install). The Starboard test runner does
-all this for you, so just use that to build and run tests. For example, to
-build and run "devel" NPLB on an ARM64 device, from the top-level directory:
+1. Check the output apk file. The output file is available at
 
-```
-starboard/tools/testing/test_runner.py -p android-arm64 -c devel -b -r -t nplb
-```
+    ```sh
+    out/android-arm_devel/nplb_evergreen_loader.apk
+    ```
 
-If you want to debug a test, you can run it from Android Studio. Edit
-`build.gradle` in the 'app' module (not to the one in the top 'apk' module) to
-change `DEFAULT_COBALT_TARGET` to be the name of the test you want to debug
-instead of 'cobalt'. Then you can set breakpoints, etc. in the test the same as
-when debugging Cobalt.
+1. To run the nplb test, execute following command
+
+    ```sh
+    # install the apk
+    adb install out/android-arm_devel/nplb_evergreen_loader.apk
+
+    # launch the apk
+    adb shell "am start --esa args '--evergreen_library=app/cobalt/lib/libnplb.so,--evergreen_content=app/cobalt/content' dev.cobalt.coat"
+    ```
+
+### Build and run nplb compat test apk for evergreen
+
+1. Build nplb compat test apk
```

Comment:
Let's use the right naming here in the title - `nplb_evergreen_compat_tests`

Reply:
`nplb_evergreen_compat_tests_install` maybe even.

Reply:
Hi @kaidokert , have a short discussion with Lam and we would like to keep `nplb_evergreen_compat_tests` as it is a target and nplb_evergreen_compat_tests_install is the build command